docker/README.md

```diff
@@ -52,7 +52,7 @@ docker-compose restart        # Restart RackHD and ELK.
 It is recommended that you uncomment the `core` RackHD service before developing RackHD. This service doesn't need to run but it should be rebuilt before other RackHD images are built.
 
 ```
-$ ../scripts/reset_submodules.bash  # Fetch latest submodules.
+$ ../scripts/reset_submodules.bash  # Fetch latest submodules, otherwise you can set every submodule to the specific commit you want.
```

Comment:
thank you.
I think this can be 
a. this reset_submodules.bash will reset the sub-modules(on-xxx) to the latest master commit of corresponding repo.
b. without running this script, the sub-modules will stay as what it is ( "linked" to a last known stable status)
c. (for advanced developer), otherwise, you can use ```git submodule update``` command to change the sub-modules "link" to any specific commit on demand.